doc/source/overview_File_Dataset_Container.rst

```diff
@@ -2,24 +2,22 @@
 Files, Datasets and Containers
 ------------------------------
 
-ATLAS has a large amount of data, which is physically stored in files. For the data management system these files are the smallest operational unit of data (so sub-file operations are not possible). Physicists need to be able to identify and operate on any arbitrary set of files.
+As data is physically stored in files, files are also the smalles operational unit of data in Rucio. Sub-file operations are currently not possible. Rucio enables users to identify and access on any arbitrary set of files.
```

Comment:
tmp_dids should maybe described in some incubation work pages ? and link to it ?

Reply:
Maybe at some later stage, I don't think it is a major concept which we have to feature that prominently. I'll put it to the todo list.